.github/workflows/build.yml

```diff
@@ -45,3 +45,4 @@ jobs:
         fi
         west twister --test sample.basic.helloworld -p siwx917_rb4338a -v --inline-logs $EXTRA_TWISTER_FLAGS
         west twister --test sample.net.wifi -p siwx917_rb4338a -v --inline-logs $EXTRA_TWISTER_FLAGS
+        west twister --test sample.bluetooth.peripheral_hr -p siwx917_rb4338a -v --inline-logs $EXTRA_TWISTER_FLAGS
```

Comment:
You need to add a `-K` flag to override the `platform_allow` list that most Bluetooth samples declare. I edited my comment after I posted it, so maybe that's why you missed it. Try running the above manually and you'll see that nothing actually gets built, but passing `-K` does make the sample build.

Reply:
I see that now thanks, will fix this.